manuscript.qmd

```diff
@@ -402,26 +417,36 @@ Absent subject matter expertise, the analyst can at least consider the time orde
 
 causal_quartet_time |>
   nest_by(dataset) |>
-  mutate(`ATE not adjusting for Z` = round(coef(lm(outcome_followup ~ exposure_baseline, data = data))[2], 2),
-         `ATE adjusting for Z` = round(coef(lm(outcome_followup ~ exposure_baseline + covariate_baseline, data = data))[2], 2)) |>
+  mutate(
+    `ATE not adjusting for Z` =
+      round_coefs(lm(outcome_followup ~ exposure_baseline, data = data)),
+    `ATE adjusting for Z` =
+      round_coefs(lm(
```

Comment:
I did some weird formatting stuff here to make sure it all fits on the page of the PDF